VMEncryption/main/Common.py

```diff
@@ -46,9 +46,13 @@ class CommonVariables:
     PassphraseFileNameKey = 'BekFileName'
     KeyEncryptionKeyURLKey = 'KeyEncryptionKeyURL'
     KeyVaultURLKey = 'KeyVaultURL'
+    KeyVaultResourceIdKey = 'KeyVaultResourceId'
+    KekVaultResourceIdKey = 'KekVaultResourceId'
     AADClientIDKey = 'AADClientID'
     AADClientCertThumbprintKey = 'AADClientCertThumbprint'
     KeyEncryptionAlgorithmKey = 'KeyEncryptionAlgorithm'
+    encryption_algorithms = ['RSA-OAEP', 'RSA-OAEP-256', 'RSA1_5']
```

Comment:
What is origin of this list?  I know they were provided as example values [here](https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/azure-disk-enc-linux#property-values) but I am not sure that list is complete.     The [SDL](https://docs.microsoft.com/en-us/azure/security/azure-security-threat-modeling-tool-cryptography#padding ) documentation lists RSA-KEM as approved but is missing from this list, and RSA 1.5 on the way out.

Reply:
Ah, well I think that's just what we supply forward to Key Vault for wrapping/unwrapping when using a kek.
According to [this](https://docs.microsoft.com/en-us/rest/api/keyvault/wrapkey/wrapkey#jsonwebkeyencryptionalgorithm) that list seems up to date.
As to what's the origin of this list? I just found it in the single pass extension and copied it over, thanks for bringing this to my attention!

Reply:
Yes I think we are good then, as we're using the most secure by default and matching the currently documented API for wrapKey.   https://docs.microsoft.com/en-us/rest/api/keyvault/wrapkey/wrapkey#jsonwebkeyencryptionalgorithm